.github/workflows/cherry-pick.yml

```diff
@@ -133,8 +133,8 @@ jobs:
         uses: rtCamp/action-slack-notify@v2
         env:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-          SLACK_TITLE: ':cherries: :x: Cherry-pick failed'
-          SLACK_MESSAGE: 'Cherry-pick [failed](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}) for [${context.payload.pull_request.title} (#${pr.number})](${context.payload.pull_request.html_url})`'
+          SLACK_TITLE: ':cherries: :x: Cherry-pick failed for “${{ github.event.pull_request.title }} (#${{ github.event.pull_request.number}})”'
+          SLACK_MESSAGE: '${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}'
```

Comment:
Title example: `🍒 ❌ Cherry-pick failed for “test(components/forms): add forms e2e testing infrastructure (#1338)”`
Message example: `https://github.com/blackbaud/skyux/actions/runs/4852956854`